AdyenComponents/PayByBank/US/PayByBankUSComponent+ConfirmationViewController.swift

```diff
@@ -0,0 +1,188 @@
+        private let model: Model
+        
+        internal lazy var headerImageView: UIImageView = {
+            UIImageView()
+        }()
+        
+        internal lazy var titleLabel: UILabel = {
+            let titleLabel = UILabel()
+            titleLabel.numberOfLines = 0
+            titleLabel.textAlignment = .center
+            return titleLabel
+        }()
+        
+        internal lazy var subtitleLabel: UILabel = {
+            let titleLabel = UILabel()
+            titleLabel.numberOfLines = 0
+            titleLabel.textAlignment = .center
+            return titleLabel
+        }()
+        
+        internal lazy var messageLabel: UILabel = {
+            let titleLabel = UILabel()
+            titleLabel.numberOfLines = 0
+            titleLabel.textAlignment = .center
+            return titleLabel
+        }()
+        
+        internal lazy var submitButton: SubmitButton = {
+            let buttonStyle = ButtonStyle(
+                title: TextStyle(font: .preferredFont(forTextStyle: .headline), color: .white),
+                cornerRounding: .fixed(8),
+                background: UIColor.Adyen.defaultBlue
+            )
+            
+            return SubmitButton(style: buttonStyle)
+        }()
+        
+        public init(model: Model) {
+            self.model = model
+            super.init(nibName: nil, bundle: nil)
+        }
+        
+        override public func viewDidLoad() {
+            super.viewDidLoad()
+            
+            headerImageView.adyen.apply(model.style.headerImage)
+            
+            titleLabel.text = model.title
+            titleLabel.adyen.apply(model.style.title)
+            
+            subtitleLabel.text = model.subtitle
+            subtitleLabel.adyen.apply(model.style.subtitle)
+            
+            messageLabel.text = model.message
+            messageLabel.adyen.apply(model.style.message)
+            
+            submitButton.title = model.submitTitle
+            submitButton.addTarget(self, action: #selector(submitTapped), for: .touchUpInside)
+            
+            let contentStack = UIStackView(arrangedSubviews: [
+                headerImageView,
+                titleLabel,
+                subtitleLabel,
+                messageLabel
+            ])
+            contentStack.spacing = 0
+            contentStack.axis = .vertical
+            contentStack.alignment = .center
+            contentStack.setCustomSpacing(10, after: subtitleLabel)
+            
+            let contentButtonStack = UIStackView(arrangedSubviews: [
+                contentStack,
+                submitButton
+            ])
+            contentButtonStack.spacing = 32
+            contentButtonStack.axis = .vertical
+            contentButtonStack.alignment = .fill
+            
+            view.addSubview(contentButtonStack)
+            contentButtonStack.adyen.anchor(inside: view.layoutMarginsGuide, with: .init(top: 16, left: 0, bottom: 8, right: 0))
```

Comment:
Ideally this should go to some style constants in the future to avoid these magic numbers and make style changes easier.